aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -59,6 +69,30 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
+template <typename input_t, typename output_t>
+void convert_indices_from_csr_to_coo_cpu(const Tensor& result, const Tensor& crow_indices, const Tensor& col_indices, const int64_t size) {
+  int64_t numel = crow_indices.numel();
```

Comment:
Nit: reading the code is easier if instead of `numel` one uses `nrows`, hence:
```suggestion
  int64_t nrows = crow_indices.numel() - 1;
```